task-1/ex1.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	var a float64
+	fmt.Scan(&a)
+	var b float64
```

Comment:
лучше объявить переменные так:
```suggestion
	var a, b float64
	fmt.Scan(&a)
```